src/globus_sdk/_remarshal.py

```diff
@@ -0,0 +1,159 @@
+"""
+This module provides internal helpers for remarshalling data from one shape
+into another.
+
+This may be as simple as converting strings from one form to another, or as
+sophisticated as building a specific internal object in a configurable way.
+"""
+
+from __future__ import annotations
+
+import collections.abc
+import typing as t
+import uuid
+
+from globus_sdk._missing import MISSING, MissingType
+from globus_sdk._types import UUIDLike
+
+T = t.TypeVar("T")
+R = t.TypeVar("R")
+
+
+def safe_strseq_iter(
+    value: t.Iterable[t.Any] | str | uuid.UUID,
+) -> t.Iterator[str]:
+    """
+    Given an Iterable (typically of strings), produce an iterator over it of strings.
```

Comment:
This docstring mis-identifies what parameters are required. Not all input types are Iterable.

```suggestion
    Iterate over one or more string/string-convertible values.
```